cmd/keeper/cmd/keeper.go

```diff
@@ -1391,6 +1480,10 @@ func (p *PostgresKeeper) postgresKeeperSM(pctx context.Context) {
 	targetRole := db.Spec.Role
 	log.Debugw("target role", "targetRole", string(targetRole))
 
+	// Set metrics to power alerts about mismatched roles
```

Comment:
These metrics are particularly useful to power an up-to-date view of keeper state. It looks like this in a dashboard:

![image](https://user-images.githubusercontent.com/3518874/57087856-e4edf500-6cf8-11e9-95d3-66fe4770de86.png)